kubernetes-tests/src/test/java/io/fabric8/kubernetes/client/mock/BindingTest.java

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright (C) 2015 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.fabric8.kubernetes.client.mock;
+
+import io.fabric8.kubernetes.api.model.Binding;
+import io.fabric8.kubernetes.api.model.BindingBuilder;
+import io.fabric8.kubernetes.client.KubernetesClient;
+import io.fabric8.kubernetes.client.server.mock.KubernetesServer;
+import org.junit.Rule;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.migrationsupport.rules.EnableRuleMigrationSupport;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@EnableRuleMigrationSupport
+class BindingTest {
+
+  @Rule
+  public KubernetesServer server = new KubernetesServer();
+
+  private KubernetesClient client;
+
+  @BeforeEach
+  void setUp() {
+    client = server.getClient().inNamespace("test");
```

Comment:
nit, I think namespace is set to `test` by default even when you use `server.getClient()`. Is this `inNamespace(..)` call required?

Reply:
Not needed at all. I took the test preparation boiler-plate from the PodTest suite and didn't even check this method.
:+1: Removing it.